_includes/project-footer.html

```diff
@@ -17,8 +17,11 @@
     </div>
     <div class="width-3-12 width-12-12-m justify-self-center align-self-center">
       <p>
-        Strimzi is licensed under the<br/>
-        <a href="https://strimzi.io/LICENSE">Apache Licence, Version 2.0</a>
+        &copy; Strimzi Authors {{ 'now' | date: "%Y" }} | Documentation distributed under <a href="https://creativecommons.org/licenses/by/4.0">CC-BY-4.0</a>
+      </p>
+
+      <p>
+        &copy; {{ 'now' | date: "%Y" }} The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage/">Trademark Usage</a> page.
```

Comment:
When I look at the preview (https://deploy-preview-146--strimzi.netlify.app/), this looks quite weird because this text is very long. I think it might be better to move the CNCF copyright and trademark usage to a new line spnning the whole site. WDYT?

Reply:
As expressed in the PR description, I was hoping that one of the maintainers who's familiar with the CSS framework could follow up and bring it in line with the site's aesthetic standards

Reply:
Ok. Did you allowed maintainers to push into this PR?

Reply:
Certainly! No problem at all.

Reply:
@scholzj Also please feel free to close this and create your own PR if that's easier.